core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -0,0 +1,30 @@
+<script type="text/ng-template" id="components/explorationFooter">
+  <div class="footer navbar-fixed-bottom oppia-exploration-footer">
+    <div class="row">
+        <div class="col-sm-7">
+          <img ng-src="<[getStaticImageUrl('/general/apple.svg')]>" itemprop="image"
+             style="height: 60px; width: 60px; float:left; margin: -25px 0px 0px 10px;"/>
```

Comment:
Drop "/" on "/>". Pull the style out into a class and place the class at the bottom of this directive. See examples in other files.